src/Charcoal/Property/PropertyField.php

```diff
@@ -247,6 +263,25 @@ public function sqlEncoding()
         return $this->sqlEncoding;
     }
 
+    /**
+     * @return array
+     */
+    public function selectExpressions()
+    {
+        return $this->selectExpressions;
+    }
+
+    /**
+     * @param mixed $selectExpressions SqlSelectExpressions for PropertyField.
+     * @return self
+     */
+    public function setSelectExpressions($selectExpressions)
+    {
+        $this->selectExpressions = $selectExpressions;
+
+        return $this;
+    }
```

Comment:
Given that a `PropertyField` is supposed to represent a single SQL column, I propose these methods explicitly represent a single column instead of a potential array of.

```suggestion
    /**
     * Retrieve the SQL SELECT field expression.
     *
     * @return string
     */
    public function sqlSelectExpression()
    {
        return $this->sqlSelectExpression ?? $this->ident();
    }

    /**
     * Set the SQL SELECT field expression.
     *
     * @param  string $expression The field expression.
     * @return self
     */
    public function setSqlSelectExpression($expression)
    {
        $this->sqlSelectExpression = $expression;

        return $this;
    }
```